pkg/modprovider/logger.go

```diff
@@ -37,15 +37,15 @@ func newComponentLogger(log pulumi.Log, resource pulumi.Resource) tfsandbox.Logg
 	return &componentLogger{log: log, resource: resource}
 }
 
-func (l *componentLogger) Log(level tfsandbox.LogLevel, message string) {
+func (l *componentLogger) Log(level tfsandbox.LogLevel, message string, ephemeral bool) {
```

Comment:
Style nit: slightly prefer separate Log and LogStatus messages, boolean params are not that readable at the call sites.